tools/sgbackstagecoverage/tools.go

```diff
@@ -0,0 +1,38 @@
+package sgbackstagecoverage
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"os"
+)
+
+// PostCoverage posts code coverage in LCOV format to the Backstage API.
+// apiURL is the URL to the API, for example 'https://api.mybackstagedeployment.com'.
+// authToken is the Bearer token for the Backstage API.
+// coverFile is the coverage file in LCOV format (use sggcov2lcov to convert Go test output).
+// entity is the entity ref, for example 'component:default/my-service'.
+func PostCoverage(ctx context.Context, apiURL, authToken, coverFile, entity string) error {
```

Comment:
We should keep in mind Go readability conventions here. With this target name and package name we'll get `sgbackstagecoverage.PostCoverage`. Maybe rename the target to just `Post`?

Reply:
Sure, good idea.